moveit_experimental/jog_arm/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+add_executable(jog_arm_server src/jog_arm/jog_arm_server.cpp)
+add_dependencies(jog_arm_server ${catkin_EXPORTED_TARGETS})
+target_link_libraries(jog_arm_server ${catkin_LIBRARIES} ${Eigen_LIBRARIES})
+
+add_executable(spacenav_to_twist src/jog_arm/teleop_examples/spacenav_to_twist.cpp)
+add_dependencies(spacenav_to_twist ${catkin_EXPORTED_TARGETS})
+target_link_libraries(spacenav_to_twist ${catkin_LIBRARIES} ${Eigen_LIBRARIES})
+
+add_executable(xbox_to_twist src/jog_arm/teleop_examples/xbox_to_twist.cpp)
+add_dependencies(xbox_to_twist ${catkin_EXPORTED_TARGETS})
+target_link_libraries(xbox_to_twist ${catkin_LIBRARIES} ${Eigen_LIBRARIES})
+
+add_executable(dragonrise_to_twist src/jog_arm/teleop_examples/dragonrise_to_twist.cpp)
```

Comment:
What is a "dragon rise?"

Reply:
It's a USB joystick somebody added via PR. I wouldn't mind removing it, though - I don't think they're common.